webpack.config.js

```diff
@@ -122,4 +122,58 @@ const webConfig = {
     },
 };
 
-module.exports = [nodeConfig, webConfig];
+/** @type fluent container scripts web worker config */
+const webWorkerConfig = {
+    mode: "none",
+    target: "webworker", // web extensions run in a webworker context
+    entry: {
+        main: "./src/web/client/common/worker/webworker.js",
+    },
+    output: {
+        filename: "[name].js",
+        path: path.join(__dirname, "./dist/web"),
```

Comment:
@amitjoshi438 this is the location where you suggested adding the webworker compiled file in the dist folder?

Reply:
better keep it in dist/common ?

Reply:
let it be in web for now